assets/arc-0031/client/utils/hooks/useEnv.ts

```diff
@@ -17,7 +19,11 @@ const createEnv = <TClient extends Record<string, ZodType>, TServer extends Reco
 
 export const useEnv = () => createEnv({
   client: {
-    algorandChainId: z.enum(['MainNet', 'TestNet', 'BetaNet'])
+    algorandChainId: (() => {
```

Comment:
Do we expect more chains apart from `MaiNet`, `TestNet` and `BetaNet`?

Reply:
I don't think so, I'm going to restore the hardcoded enum values

Reply:
Addressed in https://github.com/deanstef/ARCs/pull/4/commits/d7921dc99077fbd6ecab720e905be60bfe27d6e3